internal/generator/vector/inputs.go

```diff
@@ -142,6 +147,8 @@ func UserDefinedAppRouting(spec *logging.ClusterLogForwarderSpec, o Options) map
 					}
 					if len(matchNS) != 0 || len(matchLabels) != 0 {
 						routeMap[input.Name] = Quote(AND(OR(matchNS...), AND(matchLabels...)))
+					} else {
+						routeMap[input.Name] = fmt.Sprintf(`'%s'`, "true")
```

Comment:
No need for printf
```suggestion
						routeMap[input.Name] = "'true'"
```

Reply:
@alanconway Updated `maxRecordsPerSecond` to be `int64`. Do we need to make changes in the enhancement doc as well?